container.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">

```

Comment:
Why you don't use bootstrap to style things?
Bootstrap 5, comes with tools that help you styling and make things looks better easily. I suggest to use them rather than styling every thing with css also there is bootstrap icons, I don't mind using font awesome but i mention bootstrap icons to help you doing the task.